src/components/Model3DViewer/stories/full.md

```diff
@@ -0,0 +1,90 @@
+# Model3DViewer
+
+<!-- STORY -->
+
+#### Description:
+
+Component uses for retrieving 3D-models, which is available for provided project.
```

Comment:
@anderscognite Could you look over the description and maybe improve it if there is anything?

Reply:
Will do!